### ISSUE TITLE
Selecting a Scene in the code editor doesn't select in the storyboard, should

### ISSUE THREAD

**Issue by maltenuhn** [concrete-utopia/utopia#1045]
**Problem**
We have a overall pretty compelling follow-me feature that moves the code editor to the selected element, for rapid editing. However, this doesn't work for Scenes, which is annoying since that's where the components are, and also because you often need to true up a Scene with its content (since we don't have autosizing yet).

**Solution**
Selecting a scene should open the storyboards file in code and scroll to it "as if Scene were a regular component".

**Questions**
Where do we put the boundary? Just outside the component, or just inside it? Not relevant for Scenes, but possibly for the div / Button here:

```
<Scene> <!-- should select Scene in storyboard.js --/>
   <div> <!-- should select that div in storyboard.js --/>
       <Button label='from-button.jsx'>Click me</Button> <!-- ¿should select Button in button.jsx? or this line in storyboard.js --/>
   </div>
</Scene>
```

**Comment by Rheeseyb** [comment]
This is coincidentally something that will be solved by the current work to replace `<Scene component={App}/>` with `<Scene><App /></Scene>` (which I can confirm because I had that exact state during part of the spiking work in prep for that)

**Comment by maltenuhn** [comment]
There was a chat about this basically confirming that

**Comment by maltenuhn** [comment]
@Rheeseyb Freebie confirmed?

**Comment by Rheeseyb** [comment]
Indeed :)

**Comment by Rheeseyb** [comment]
Closed by #1083
